docusaurus/docs/jaqpotpy/scikit-learn-models/feature-preprocessing.md

```diff
@@ -5,11 +5,33 @@ sidebar_position: 4
 
 ## Using Multiple Featurizers
 
-In the first script, we initialize two molecular descriptor calculators from JaqpotPy:
+This guide is about using multiple featurizers and performing feature selection.
+
+First, we import necessary libraries.
 
 ```python
+# Import necessary libraries
+import pandas as pd
+import numpy as np
+from sklearn.feature_selection import VarianceThreshold
+from sklearn.compose import ColumnTransformer
+from sklearn.preprocessing import MinMaxScaler, OneHotEncoder, StandardScaler
+from sklearn.ensemble import RandomForestRegressor
+from jaqpotpy.models import SklearnModel
+from jaqpotpy.datasets import JaqpotpyDataset
 from jaqpotpy.descriptors import RDKitDescriptors, MACCSKeysFingerprint
+```
+
+Create a dataframe with SMILES strings, a categorical variable, temperature, and activity values
```

Comment:
Can we have a link to the SMILES wiki page or smth on the docs so that someone who sees this example and not know what SMILES is can link maybe?

Reply:
Updated!